modules/Kconfig.mcux

```diff
@@ -229,6 +229,12 @@ config HAS_MCUX_TRNG
 	  Set if the true random number generator (TRNG) module is present in
 	  the SoC.
 
+config HAS_MCUX_CAAM
```

Comment:
The `HAS_MCUX_CAMM` symbol should not be needed with the `DT_HAS` method of enabling device drivers